pandas/tests/plotting/test_series.py

```diff
@@ -729,6 +729,26 @@ def test_dup_datetime_index_plot(self):
         s = Series(values, index=index)
         _check_plot_works(s.plot)
 
+    def test_errorbar_asymmetrical(self):
+        # GH9536
+        s = Series(np.arange(10), name="x")
+        err = np.random.rand(2, 10)
+
+        ax = s.plot(yerr=err, xerr=err)
+
+        result = np.vstack([i.vertices[:, 1] for i in ax.collections[1].get_paths()])
+        expected = (err.T * np.array([-1, 1])) + list(zip(s, s))
```

Comment:
IMO the last operand would clearer as
```python
s.to_numpy().reshape(-1, 1)
```
, but it's up to you

Reply:
Updated in https://github.com/pandas-dev/pandas/pull/34514/commits/67b534d26acd5c722e561c3739a4ee91a6550736